program-runtime/src/sysvar_cache.rs

```diff
@@ -231,3 +232,86 @@ pub mod get_sysvar_with_account_check {
         invoke_context.get_sysvar_cache().get_stake_history()
     }
 }
+
+pub mod get_sysvar_with_account_check2 {
```

Comment:
Why named "2" and not just replace `get_sysvar_with_account_check`?

Reply:
Because I want to change each builtin program in a separate PR.
And yes, at the end I will remove the "2" and delete the `KeyedAccount` version.